### DISCUSSION TITLE
State of collections during snapshot restore

### DISCUSSION THREAD

**Discussion by robertmacyiii** [qdrant/qdrant#3992] [Q&A]
I am considering using snapshot restores periodically on a single-node instance of qdrant, but I want the collections to still be queried during the restore process. During a snapshot restore operation on a node, are reads possible on that collection? If so, will they be reading the old data on the collection or the in-progress loaded data from the snapshot of the collection, or would the snapshot restore be updating only the points that are different?

**Comment by timvisee** [comment]
> During a snapshot restore operation on a node, are reads possible on that collection?

No, if I recall correctly. But I'd have to dive into the code to be sure.

On a multi-node cluster this should work fine, because you'd be able to restore one by one in that case.
